src/app/ApplicationTemplate.UWP/Views/Content/DadJokes/DadJokesPage.xaml

```diff
@@ -59,13 +59,14 @@
 							  not_android:ItemsSource="{Binding Data}">
 						<ListView.ItemTemplate>
 							<DataTemplate>
-								<Grid Background="{StaticResource MaterialSurfaceBrush}">
+								<Grid Background="{StaticResource MaterialSurfaceBrush}"
+									  Margin="0,10">
 									<Grid.ColumnDefinitions>
 										<ColumnDefinition Width="*" />
 										<ColumnDefinition Width="Auto" />
 									</Grid.ColumnDefinitions>
 
-									<StackPanel Margin="0,16">
+									<StackPanel Padding="15">
```

Comment:
Aren't we using multiples of 4 for everything margin and padding? 
i.e. 4, 8, 12, 16, 20, 24, etc.

Reply:
That was a reflex of mine, even on the design it's 16. I will put 16

Reply:
Same comment just above where there's a 10 (that should be 8 or 12 I assume)